src/ResourceManager/Sql/Commands.Sql/Auditing/Model/AuditingPolicyModel.cs

```diff
@@ -25,9 +25,10 @@ public enum StorageKeyKind { Primary, Secondary };
     public enum AuditStateType { Enabled, Disabled, New };
 
     /// <summary>
-    /// The possible value of auditing type
+    /// The possible values of auditing type. 
+    /// NotSet is a temporary value in order to mark that AuditType parameter was not provided at all by the user. It is only initial value, and won't be returned to the user.
     /// </summary>
-    public enum AuditType { Table, Blob };
+    public enum AuditType { NotSet, Table, Blob };
```

Comment:
@ayeletshpigelman possible breaking change, if user has used enum values instead of enums.
Would be better if the new value is added at the end (after Blob)